### PR TITLE
fix(core): `PrimitiveTextfield` long custom content overflow

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [X] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?
before

## What is the new behavior?
after

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: projects/demo/src/modules/components/primitive-textfield/primitive-textfield.component.ts [hunk 1]
```diff
@@ -40,7 +21,7 @@ viewBox="0 0 24 24">
         },
     ],
 })
-export class ExampleTuiPrimitiveTextfieldComponent extends AbstractExampleTuiInteractive {
+export class ExampleTuiPrimitiveTextfieldComponent extends AbstractExampleTuiControl {
```
Comment (vladimirpotekhin) [inline projects/demo/src/modules/components/primitive-textfield/primitive-textfield.component.ts:24]: PrimitiveTextfield hasn't pseudoInvalid property (inherited from TuiReactiveControl) :pepe:

Comment (nsbarsukov) [reply projects/demo/src/modules/components/primitive-textfield/primitive-textfield.component.ts:24]: oops

Comment (nsbarsukov) [reply projects/demo/src/modules/components/primitive-textfield/primitive-textfield.component.ts]: fixed

File: projects/core/components/primitive-textfield/primitive-textfield.style.less [hunk 2]
```diff
@@ -13,7 +13,7 @@
     display: flex;
     align-items: center;
     justify-content: center;
-    width: 2rem;
+    min-width: 2rem;
```
Comment (nsbarsukov) [inline projects/core/components/primitive-textfield/primitive-textfield.style.less:16]: hm... i thing extra screenshot test will not be redundant.

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/core/components/primitive-textfield/primitive-textfield.style.less (hunk 2 above)
Comment (nsbarsukov) [reply projects/core/components/primitive-textfield/primitive-textfield.style.less:16]: done

**Review by waterplea — APPROVED** [review]

**Pull request merged**